Test RegEx not catching all cases

Please describe your issue clearly and also include all details below.

**Details**

- _avn_ `0.2.3`
- _node_ `v6.11.5`
- _nvm_ `0.33.6`
- _n_ `2.0.0`
- _zsh_ `zsh 5.0.8 (x86_64-apple-darwin15.0)`
- _bash_ `version 3.2.57(1)-release (x86_64-apple-darwin15)`

The output of `__avn_debug` in the directory with a `.node-version` file is:

```sh
# paste output here
```

`avn` is loaded in my `~/.{bash|zsh}{_profile|rc}` file with:

```sh
# find the relevant line(s) in your profile & note path above
```

**nvm specific**

- [x] As an `nvm` user I am confirming that I did _not_ install with Homebrew

I am starting in on a PR for #43, but I have a couple of tests failing. One of them is the `fails for any failed action` test in `test/setup.js`, which give the following failed test output:

```bash
AssertionError: expected 'error: EACCES: permission denied, open \'/var/folders/1p/9z6dp_4n2cd27d2ktj6w_xv40000gn/T/d-1171011-15092-10e8ilz.4tsh8h0k9/.bash_profile\'\n' to match /^error: EACCES.*, open '[\/\w-]*\/.bash_profile'\n$/
```

It looks like the regex is just not catching my error message case. I can dig into the regex and fix it to catch this, but I wanted to bring this to your attention in case it's a known issue that I should ignore.

--- a/test/setup.js
+++ b/test/setup.js
@@ -371,7 +371,7 @@ describe('avn setup', function() {
         'avn: configuration complete (~/.avnrc)',
         'avn: installation complete',
       ]);
-      expect(std.err).to.match(/^error: EACCES.*, open '[\/\w-]*\/.bash_profile'\n$/);
+      expect(std.err).to.match(/^error: EACCES.*, open '[\/\w-\d\.]*\/.bash_profile'\n$/);
     });
   });
 });

